frontend/src/app/components/channel/small/small.controller.ts

```diff
@@ -251,4 +251,12 @@ export class SmallChannelController extends BaseChannelController {
         attachMediaStream(slotElement, stream);
       });
   }
+  private equalComposites(first: IUser[] , second: IUser[]  ){
```

Comment:
`is` and `are` are the same verb :-)
nevermind, we can leave it as is also

On Wed, May 4, 2016 at 10:59 AM, davgur <notifications@github.com> wrote:

> In frontend/src/app/components/channel/small/small.controller.ts
> <https://github.com/Bnei-Baruch/galaxy2/pull/49#discussion_r62008104>:
>
> > @@ -251,4 +251,12 @@ export class SmallChannelController extends BaseChannelController {
> >          attachMediaStream(slotElement, stream);
> >        });
> >    }
> > +  private equalComposites(first: IUser[] , second: IUser[]  ){
>
> isCompositesAreEqual is better?
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/Bnei-Baruch/galaxy2/pull/49/files/c1d158369b5d4c963541a19cefc23e1b061ba070#r62008104>
>



-- 
Kind regards,
Konstantin Alexandrov

http://illuzia.net